guvnor-rest/guvnor-rest-backend/src/main/java/org/guvnor/rest/backend/ProjectResource.java

```diff
@@ -275,27 +275,26 @@ public Response createProject(
         logger.info( "-----getProjects--- , repositoryName: {}", repositoryName );
 
         Repository repository = repositoryService.getRepository(repositoryName);
-        if( repository == null ) { 
+        if( repository == null ) {
             throw new WebApplicationException( Response.status( Response.Status.NOT_FOUND ).entity( repositoryName ).build() );
         }
-        
+
         Set<Project> projects = projectService.getProjects(repository, repository.getCurrentBranch());
-        
+
         List<ProjectResponse> projectRequests = new ArrayList<ProjectResponse>(projects.size());
-        for( Project project : projects ) { 
+        for( Project project : projects ) {
            ProjectResponse projectReq = new ProjectResponse();
            GAV projectGAV = project.getPom().getGav();
            projectReq.setGroupId(projectGAV.getGroupId());
-           projectReq.setName(projectGAV.getArtifactId());
```

Comment:
@psiroky. No. Please always first take a look at the actual code before making comments... :/ 

See the following classes: 

- [ProjectResponse](https://github.com/droolsjbpm/guvnor/blob/master/guvnor-rest/guvnor-rest-client/src/main/java/org/guvnor/rest/client/ProjectResponse.java)
- [ProjectRequest](https://github.com/droolsjbpm/guvnor/blob/master/guvnor-rest/guvnor-rest-client/src/main/java/org/guvnor/rest/client/ProjectRequest.java)
- [Entity](https://github.com/droolsjbpm/guvnor/blob/master/guvnor-rest/guvnor-rest-client/src/main/java/org/guvnor/rest/client/Entity.java)